src/assets/data/process.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "imageId": 70764,
+    "relatedImages": [
+      { "tag": "Cat", "imageId": 70755 },
+      { "tag": "Dog", "imageId": 70732 },
+      { "tag": "Fish", "imageId": 70678 }
+    ]
+  }
+]
```

Comment:
probably easier to use, else you will have to parse all elements to find the image for the next selected one:
```
[
  "70764":  [
      { "tag": "Cat", "imageId": 70755 },
      { "tag": "Dog", "imageId": 70732 },
      { "tag": "Fish", "imageId": 70678 }
    ]
]
```

Reply:
Right for the next part, it is better